src/CoreBundle/private/modules/MTTableManager/MTTableManager.class.php

```diff
@@ -415,17 +415,15 @@ public function GetAutoCompleteAjaxList()
         while ($record = $recordList->Next()) {
             $name = $record->GetName();
             if (!empty($name)) {
-                $tmp = new stdClass();
+                $html = $name;
                 if ($record->id == $recordID) {
-                    $name = $name.' <---';
+                    $html = '<strong>'.$name.'</strong>';
                 }
-                $tmp->label = $name;
-                $tmp->value = $record->id;
-                $returnVal[] = $tmp;
+                $returnVal[] = ['id' => $record->id, 'text' => $name, 'html' => $html];
             }
         }
 
-        return $returnVal;
+        return json_encode($returnVal);
```

Comment:
We need to do this change because the library that is used to retrieve the information needs JSON now. This method is public to allow it to call it via ajax, but it is only used by the backend JS for record switching which was replaced at the same time. So this is combined and doesn't create a BC break.